oneplus/misc_views.py

```diff
@@ -118,7 +127,7 @@ def get():
         state['fname'] = ""
         state['sname'] = ""
         state['comment'] = ""
-        state['contact'] = ""
+        state['contact'] = "" if not usr or not usr.mobile else usr.mobile
```

Comment:
That's similar to a ternary operator in Java. This variable isn't allowed to be `None`, so the code is just replacing it with an empty string if the object or the field are `None`.